examples/simple-react/src/lib/youbet-sdk/options.ts

```diff
@@ -0,0 +1,19 @@
+import { SdkCtorOptions } from "youbet-sdk";
+
+export const lineaOptions: SdkCtorOptions = {
```

Comment:
Better to name with lineaTestOptions? But I suggest you to checkout other multichain sdk implementation first. And reply this comment with conculusion and links. If you can write a document in our feishu space is the best.

Reply:
But actually it's not a test network?

you can see other multichain sdk implementation here:
https://web3js.readthedocs.io/en/v1.10.0/web3-eth-contract.html
https://web3js.readthedocs.io/en/v1.10.0/web3.html#example-remote-node-provider
https://docs.ethers.org/v5/api/contract/contract/#Contract--creating
https://docs.ethers.org/v5/api/providers/jsonrpc-provider/#JsonRpcProvider